### DISCUSSION TITLE
demo license

### DISCUSSION THREAD

**Discussion by hiddify-com** [dashingsoft/pyarmor#1018] [Ideas]
I have currently a project.

I want to test obfs with version 8. Of course i want to buy your great product but i want to test if it is applicable with my current project.
Is it possible to have 7 days demo?

**Comment by jondy** [comment]
Sorry, Pyarmor is a small tool and only cost small money, there is no demo license plan.

Most of features could be verified in trial version, other advanced features, for example, mix-str, bcc mode and rft mode, could be configured to ignore one function or one script so that all the others could work with these advanced features.

**Comment by hiddify-com** [comment]
Great thanks
If it is possible to obfs a submodule from my package, would you please let me know how to do it? because version 8.0 documentation is so small.

Thank you

**Comment by jondy** [comment]
Specify only one submodule in the command line to obfuscated all the files in this path,  you can mix the obfuscated script and plain scripts.
